# sets the active degrees of freedom of the robot (used by planners to get configuration space)
string sessionname
int32 sessionid

int32 bodyid
ActiveDOFs active
---
